Source/WebCore/page/EventHandler.cpp

```diff
@@ -1459,7 +1459,7 @@ std::optional<Cursor> EventHandler::selectCursor(const HitTestResult& result, bo
         return std::nullopt;
 
     auto renderer = node->renderer();
```

Comment:
Can you move this variable further down closer to where it's used? (also it should be `auto*`)